x-pack/platform/plugins/shared/alerting/server/plugin.ts

```diff
@@ -731,6 +731,18 @@ export class AlertingPlugin {
           const [, { security }] = await core.getStartServices();
           return security?.authc.apiKeys.areAPIKeysEnabled() ?? false;
         },
+        hasRequiredPrivilegeGrantedInAllSpaces: async ({
```

Comment:
I am skeptical of exposing this from the route's context. All plugins will have access to. If it is only for internal usage, could we create a util function that the router will use?

Reply:
@cnasikas I couldn't find another way to have the `core` object in a route. Wdyt about [aa462a0](https://github.com/elastic/kibana/pull/217226/commits/aa462a05cfad8080e564968ef3922819a90c2743)?